lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1957,6 +1957,108 @@ def plot(self, xs, ys, *args, zdir='z', **kwargs):
 
     plot3D = plot
 
+    def fill_between(self, x1, y1, z1, x2, y2, z2, *,
+                     where=None, mode='auto', **kwargs):
+        """
+        Fill the area between two 3D curves.
+
+        The curves are defined by the points (*x1*, *y1*, *z1*) and
+        (*x2*, *y2*, *z2*). This creates one or multiple quadrangle
+        polygons that are filled. All points must be the same length N, or a
+        single value to be used for all points.
+
+        Parameters
+        ----------
+        x1, y1, z1 : float or 1D array-like
+            x, y, and z  coordinates of vertices for 1st line.
+
+        x2, y2, z2 : float or 1D array-like
+            x, y, and z coordinates of vertices for 2nd line.
+
+        where : array of bool (length N), optional
+            Define *where* to exclude some regions from being filled. The
+            filled regions are defined by the coordinates ``pts[where]``,
+            for all x, y, and z pts. More precisely, fill between ``pts[i]``
+            and ``pts[i+1]`` if ``where[i] and where[i+1]``. Note that this
+            definition implies that an isolated *True* value between two
+            *False* values in *where* will not result in filling. Both sides of
+            the *True* position remain unfilled due to the adjacent *False*
+            values.
+
+        mode : {'auto', 'quad', 'polygon'}, default: 'auto'
+            The fill mode.
+            If 'auto' and all the coordinates for the x, y, or z input pairs
+            are the same, then the points lie on a plane and 'polygon' is used.
+            Otherwise, 'auto' defaults to 'quad'.
+            If 'quad', then a separate quadrilateral polygon is created for
+            each pair of subsequent points in the 1st and 2nd lines. This is
+            more flexible, but may be slower and result in rendering artifacts.
+            If 'polygon', then the 1st and 2nd lines are connected to form a
+            single polygon, which may be faster and render more cleanly for
+            simple shapes (eg, for filling between two lines that lie on a
+            single plane).
```

Comment:
```suggestion
            The fill mode. One of:

            - 'quad':  A separate quadrilateral polygon is created for each
              pair of subsequent points in the two lines.
            - 'polygon': The two lines are connected to fore a single polygon.
              This is faster and can render more cleanly for simple shapes (e.g.
              for filling between two lines that lie within a plane).
            - 'auto': If the lines are in a plane parallel to a coordinate axis
              (one of *x*, *y*. *z* for both lines is a constant), 'polygon' is
              used. Otherwise, 'quad' is used.
```

Reply:
> one of *x*, *y*. *z* for both lines is a constant

They need to be constant and equal for this condition to be true, see for a counterexample:

```
n = 50
theta = np.linspace(0, 2*np.pi, n)
x1 = np.cos(theta)
y1 = np.sin(theta)
z1 = 0
x2 = np.cos(theta + np.pi)
y2 = np.sin(theta + np.pi)
z2 = 1
```
![image](https://github.com/matplotlib/matplotlib/assets/14363975/bc60bc3a-044d-4990-ab2a-139172e2d178)

Updated this docstring to match your format.